src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py

```diff
@@ -43,6 +43,20 @@ def test_plot_calls_plt_show(mock_show):
     assert cl._attractor_present
 
 
+@mock.patch("einsteinpy.plotting.geodesics_static.plt.show")
+def test_plot_calls_plot_show(mock_animated_plt_show):
```

Comment:
Not a good test! The test should check and assert values related to animate and not show :)

Reply:
Yes, I initially added the `show` method to `animate` and changed it later as suggested in #177  but forgot changing the tests. I'll make the changes ASAP.